### PR TITLE
Added html and css to style guide

### PR DESCRIPTION
**Pull request by ghost**
@cfnelson @karldanninger

### CODE REVIEW

File: style-guide/code-style/sass/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Formatting
+* Use SCSS syntax.
+* Use a space between CSS property and its value, `<property>: <value>`.
+* Use a space between CSS selector and code block, `selector {}`.
+* Use a new line after the comma when listing selectors.
+* Use hex values for colors when possible.
+* Don't specify a unit after using a 0 value.
+* Avoid nesting more than 2 levels, unless pseudo classes.
+* Use
```
Comment (cfnelson) [inline style-guide/code-style/sass/README.md]: "Use" ???

File: style-guide/code-style/html/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# HTML 
+
+## Formatting
+* Don't use inline styles, unless you are generating dynamic styles in your templating system.
```
Comment (cfnelson) [inline style-guide/code-style/html/README.md]: Some Inlining of styles is recommended or advocated for performance. e.g - displaying above the fold content. Check [here](https://developers.google.com/speed/docs/insights/OptimizeCSSDelivery) and let me know what you think ? Can maybe add some reference or content from this ?

**Review by cfnelson — COMMENTED** [review]
Out of curiosity have you looked at or used [css-modules](https://github.com/css-modules/css-modules) or [radium](https://github.com/FormidableLabs/radium) though radium is react specific. Kind of like css-modules as it isn't framework dependent.

**Comment by cfnelson** [comment]
Also, I do not know enough on HTML/CSS to give a strong/assertive opinion on best style/practices. Should get @karldanninger in on this review :) Nice work otherwise :)

**Comment by karldanninger** [comment]
Hehe, I might start advocating we write all of our styles in JavaScript

File: style-guide/code-style/html/README.md (hunk 2 above)
Comment (pauldowman) [reply style-guide/code-style/html/README.md]: We should always start with the most maintainable thing until/unless there's a specific need for some sort of gnarly optimization. I think @oliverthomasklein is trying to write general guidelines.

Comment (karldanninger) [reply style-guide/code-style/html/README.md]: Agreed

File: style-guide/code-style/html/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+* Only use ID's to be targeted my JS.
```
Comment (pauldowman) [inline style-guide/code-style/html/README.md]: _by_ JS

**Comment by pauldowman** [comment]
Instead of a sass style guide should it just be for CSS in general?

**Comment by ghost** [comment]
@pauldowman I was pretty unsure on this. A lot of the formatting is redundant if you are using CSS. I looked at how Thoughtbot did it, they used sass. I think it's more accurate.

File: style-guide/code-style/html/README.md (hunk 2 above)
Comment (ghost) [reply style-guide/code-style/html/README.md]: 👍

**Comment by cfnelson** [comment]
@oliverthomasklein @karldanninger This has been sitting around for quite some time. So long that we are now using Styled Components. 😂  

Would love to get this refactored and merged in. Happy to help and provide my perspective as well to help push this along.

Some suggestions:
- Don't delete the `Readme.md`, the files seem short enough to exist in the one file under their own headings! 💯  If it get's large we can add a table of contents. 🍡 
- We should add a section for `CSS in JSS` 🎋  e.g - `styled-components`, `emotion`, etc.
- `SASS` should become `CSS Preprocessors (less/sass/stylus)` ..etc? 🥇  
-  Or `SASS` could become a sub-heading to `CSS Prepocessors` if you wanted specific recommendations instead of generalised ones?

Going to pull in @xavcz as well as I am sure he will have/add some 🔥  and insight. Would you be interested in giving any thoughts or dot points xavs? Will be happy to type/clean them up for you as I know you are extremely busy. 👷

**Comment by xavxyz** [comment]
🤐 no opinion. Use whatever fits the job, the project structure & the team: don't be dogmatic, there is no 1 solution fit all.

**Comment by cfnelson** [comment]
@xavcz I might have not communicated the purpose of this PR properly. Sorry 😞 . This section of the OK GROW! guides is our `Style Guide`. It states our opinion on best practices for how to structure/format/design your app when writing html/css/(css preprocessor/css in jss) etc. 

Even your comment of `don't be dogmatic, there is no 1 solution fit all` is a good point to make and should be mentioned somewhere. 🎉  😄

**Comment by xavxyz** [comment]
Closing, but content should be re-used!

**Pull request closed without merging**